### PR TITLE
Jalexcrawford basic UI

### PR DESCRIPTION
**Pull request by jalexcrawford**
Basic layout for the main game UI.

The buttons are functional, but need to be wired to the contract and employee system.

The layout is done, but need to generate the main game script to update the resource counters.

### CODE REVIEW

File: Assets/MainWindow.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MainWindow : MonoBehaviour
```
Comment (andrewcgraves) [inline Assets/MainWindow.cs:5]: Maybe we don't need this file until we create a story for this

File: Assets/Scripts/GameManager/GameManager.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    // Start is called before the first frame update
```
Comment (andrewcgraves) [inline Assets/Scripts/GameManager/GameManager.cs:7]: Same comment for this one as last

**Review by andrewcgraves — CHANGES_REQUESTED** [review]

File: Assets/MainWindow.cs (hunk 1 above)
Comment (jalexcrawford) [reply Assets/MainWindow.cs]: oops, got too involved in the tutorial. :)

File: Assets/Scripts/GameManager/GameManager.cs (hunk 2 above)
Comment (jalexcrawford) [reply Assets/Scripts/GameManager/GameManager.cs]: also removed, part of the tutorial that isn't necessary  for this request.

File: Assets/Scripts/Windows/GenericWinow.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class GenericWinow : MonoBehaviour
```
Comment (johnavalos2) [inline Assets/Scripts/Windows/GenericWinow.cs:6]: Nitpick: typo for "GenericWinow" -> "GenericWindow"

**Review by andrewcgraves — APPROVED** [review]

File: Assets/Scripts/Windows/GenericWinow.cs (hunk 3 above)
Comment (jalexcrawford) [reply Assets/Scripts/Windows/GenericWinow.cs:6]: It builds though! :P

**Review by johnavalos2 — APPROVED** [review]

File: Assets/Scripts/Windows/GenericWinow.cs (hunk 3 above)
Comment (Frances637) [reply Assets/Scripts/Windows/GenericWinow.cs:6]: I still suggest to fix spelling errors to make it look professional.

**Review by Frances637 — COMMENTED** [review]
Other than a couple typos, the scripts for the basic UI look functional so far.

**Pull request merged**